docker/Dockerfile

```diff
@@ -18,67 +18,55 @@
 FROM centos:7 AS builder
 
 # install dependencies
-RUN yum makecache && yum -y update && yum -y groupinstall 'Development Tools' && \
-    yum install -y byacc automake java-11-openjdk-devel java-1.8.0-openjdk-devel libtool bison binutils-devel zip \
-    unzip ncurses-devel curl git wget python2 glibc-static java-1.8.0-openjdk-devel ccache \
-    libstdc++-static which psl libpsl-devel centos-release-scl && \
-    yum install -y devtoolset-10 devtoolset-10-gcc devtoolset-10-libubsan-devel devtoolset-10-liblsan-devel \
-    devtoolset-10-libasan-devel
-
-# build cmake
-ARG CMAKE_VERSION=3.22.1
-ARG CMAKE_BASE_URL=https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}
-RUN wget ${CMAKE_BASE_URL}/cmake-${CMAKE_VERSION}-Linux-x86_64.sh -q -O /tmp/cmake-install.sh && \
-    chmod u+x /tmp/cmake-install.sh && \
-    /tmp/cmake-install.sh --skip-license --prefix=/usr --exclude-subdir && \
-    rm /tmp/cmake-install.sh
-
-# build  ninja
-ARG NINJA_VER=1.10.2
-ARG NINJA_BASE_URL=https://github.com/ninja-build/ninja/releases/download/v${NINJA_VER}
-RUN wget -q ${NINJA_BASE_URL}/ninja-linux.zip -O /tmp/ninja-linux.zip && \
-    unzip /tmp/ninja-linux.zip -d /usr/bin/ && \
-    rm /tmp/ninja-linux.zip
+RUN yum install -y byacc patch automake libtool make which file ncurses-devel gettext-devel unzip bzip2 zip util-linux wget git python2 \
+    java-11-openjdk-devel java-1.8.0-openjdk-devel
+
+# install ninjia
+RUN wget -q -O /tmp/ninja-linux.zip https://doris-thirdparty-repo.bj.bcebos.com/thirdparty/ninja-linux.zip \
+    && unzip /tmp/ninja-linux.zip -d /usr/bin/ \
+    && rm /tmp/ninja-linux.zip
+
+# install autoconf-2.69
+RUN wget https://doris-thirdparty-repo.bj.bcebos.com/thirdparty/autoconf-2.69.tar.gz && \
```

Comment:
If the image is based on centos 7, the system's autoconf and bison can be used directly with `yum install bison`, (autoconf should already be installed as a dependency of other packages, likely automake)

Reply:
Remove